css/base/_colors.scss

```diff
@@ -0,0 +1,8 @@
+$color-white: #ffffff;
+$color-black: #000000;
+$color-primary: #dd2a7c;
+$color-secondary: #3095f6;
+$color-alpha: #fafafa;
+$color-bravo: #efefef;
+$color-charlie: #dbdbdb;
```

Comment:
jesli chodzi o liczebniki to uzywamy raczej tylko primary, secondary, tertiary

natomiast pozniej trzeba kombinowac i np szukac podobienstw i isc w kierunku  secondary--light, secondary--dark itp jesli sa to np rozne odcienie szarego albo innego koloru

alfa, bravo, charlie wogole nie sugeruje co to za kolor, albo do czego jest uzywany, bo primary i secondary to wiadomo ze sa to kolory glowne, wiec mozna sie domyslic jakie to sa znajac projekt

tu sa ciekawe przyklady:

https://css-tricks.com/what-do-you-name-color-variables/

np colorNeutral